src/olympia/editors/templates/editors/queue.html

```diff
@@ -162,13 +162,7 @@
             {% for value in row %}
               <td>{{ value|xssafe }}</td>
             {% endfor %}
-            {% if table.show_version_notes %}
-            <td>
-              <div class="addon-version-notes">
-                <a href="#" class="ed-sprite-notes app-icon" title="{{ _('Version Notes / Notes for Reviewer') }}">&nbsp;</a>
-              </div>
-            </td>
-            {% endif %}
+            <td>&nbsp;</td>
```

Comment:
Current prod looks like:

<img width="136" alt="screen shot 2016-10-24 at 23 44 34" src="https://cloud.githubusercontent.com/assets/873258/19665152/edcc9346-9a43-11e6-88c7-6dd17d355000.png">

That tiny white space near the border comes from https://github.com/mozilla/addons-server/blob/master/static/css/zamboni/zamboni.css#L2-L21, so I'm adding a fake column to fill the row.